### PR TITLE
Implemented mock test for data_scraping

### PR DESCRIPTION
**Pull request by fabiansd**
An attempt to implement unittesting with mocking

### CODE REVIEW

File: test/data_scraping/test__main__.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import unittest
+from unittest.mock import patch, Mock
```
Comment (loeiten) [inline test/data_scraping/test__main__.py]:
```suggestion
from unittest.mock import patch
from unittest.mock import Mock
```

**Review by loeiten — CHANGES_REQUESTED** [review]
Please remove: 
    - `chromedriver.exe` (you can also put `*.exe` in `.gitignore`).
    - `12. banana_PNG838.png`

Please do not remove:
    - `2. banana-2`

File: test/data_scraping/test__main__.py (hunk 1 above)
Comment (fabiansd) [reply test/data_scraping/test__main__.py]: Ignoring .exe files written in -gitignore and chromedriver.exe removed from repository.

I fo not know what you mean by the banana image files however. Those are ignored by the gitignore file. How do you even know what banana pictures I keep in my banana image folder locally?

Comment (fabiansd) [reply test/data_scraping/test__main__.py]: I have looked into the case. The change is necessary. The reason is that an image in the master repository is corrupted and cannot be opened. It is not a jpg file.

The bad file was replaced by a good file :)

Comment (loeiten) [reply test/data_scraping/test__main__.py]: The corrupted file is there to test some functionality. I'll fix :)

**Review by loeiten — APPROVED** [review]

**Pull request merged**